sponge.go

```diff
@@ -180,76 +194,97 @@ type NytList struct {
 	Results []NytItem `json:"results"`
 }
 
-func getNyt() []Formatted {
+func getNyt() ([]Formatted, error) {
 	apiKeyName := "NYT_API_KEY"
-
-	nytApiKey := os.Getenv(apiKeyName)
-	if nytApiKey == "" {
-		fmt.Printf("Env var %s not found!\n", apiKeyName)
-		return make([]Formatted, 0)
+	nytApiKey, apiKeyErr := getEnvVar(apiKeyName)
```

Comment:
now that you've refactored the code to a single function, not much point in having a variable just to hold a string literal.